Makefile

```diff
@@ -11,8 +11,22 @@ PDF_GRAMMARLY = grammarly.pdf
 
 IMAGES := $(wildcard images/*.jpg images/*.pdf images/*.png)
 
+# spag is a phony rule, it generates index.html
+# from textidote
+.PHONY: spag
+
+
 all: ${PDF_PAPER}
 
+# spelling and grammar
+spag: paper.tex
```

Comment:
Do you think `grammar` would be better?
I 100% know I would forget `spag` when trying to remember the command line name.